iq_geotree.install

```diff
@@ -42,7 +44,8 @@ function iq_geotree_create_config() {
       // Create a new vocabulary from the yaml configuration and save.
       $taxonomy_vocabulary_config = $default_config_storage->read('taxonomy.vocabulary.' . $vid);
       if (empty($taxonomy_vocabulary_config)) {
-        // If config-sync not available then read module's config for vocabulary.
+        // If config-sync not available then read module's config
```

Comment:
I am very picky here, but could you use the multiline comments syntax?
https://www.php.net/manual/en/language.basic-syntax.comments.php
 :-)